src/components/ParserComponent.tsx

```diff
@@ -16,354 +16,354 @@ import 'bootstrap/dist/css/bootstrap.min.css';
  */
 export default class ParserComponent extends React.Component<ParserInterface,
   ParserState> {
-    private columnTypes?: Array<Column>;
```

Comment:
Why did you reformat this entire file? I'd revert this change.

Reply:
Sorry, I had a bit of a smooth brain moment and instead of reverting it i just copy & pasted develop's ParserComponent.tsx 😅 Forgive me 🙏